belt-dwp/README.md

```diff
@@ -0,0 +1,59 @@
+# RustCrypto: BeltDwp
+
+[![crate][crate-image]][crate-link]
+[![Docs][docs-image]][docs-link]
+![Apache2/MIT licensed][license-image]
+![Rust Version][rustc-image]
+[![Project Chat][chat-image]][chat-link]
+[![Build Status][build-image]][build-link]
+
+Pure Rust implementation of **BeltDwp** ([STB 34.101.31-2020][1]): an
+[Authenticated Encryption with Associated Data (AEAD)][2].
+
+[Documentation][docs-link]
```

Comment:
Since you use readme as crate docs, this doc link is not needed, since it looks strange in generated crate docs.